ts-core/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "3.9.0",
   "description": "Javascript SDK for differential.dev",
   "main": "bin/index.js",
+  "bin": "bin/cli/index.js",
```

Comment:
I have included this in `ts-core` for convenience. I.e a differential service can simply add `deploy: differential deploy ...` to `package.json#scripts`.

However, this would mean services will pull in a bunch of unnecessary packages.